src/__tests__/http-test.ts

```diff
@@ -96,11 +95,11 @@ describe('GraphQL-HTTP tests for connect', () => {
 
     return {
       request: () => supertest(app),
-      use: (...args) => app.use(...args),
+      use: app.use.bind(app),
```

Comment:
Tweaked these functions to get around TS complaining about wrong number of arguments.